doc/reference/reference_lua/socket.rst

```diff
@@ -815,11 +835,11 @@ On the first shell, start Tarantool and say:
     }
     )
 
-The above code means: use `tcp_server()` to wait for a
-connection from any host on port 3302. When it happens,
-enter a loop that reads on the socket and prints what it
+The above code means: |br|
```

Comment:
Maybe we'd better present these steps as a numbered list?

Reply:
fixed in e64c567